lib/rules/background-color-validation/index.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * @author Aleksandra Borowska <https://github.com/Ola2808-Boro>
+ */
+'use strict';
+
+const report = require('../../utils/report');
+const ruleMessages = require('../../utils/ruleMessages');
+const validateOptions = require('../../utils/validateOptions');
+const isStandardSyntaxDeclaration = require('../../utils/isStandardSyntaxDeclaration');
+const ruleName = 'background-color-validation';
+
+const messages = ruleMessages(ruleName, {
+	rejected:
+		`Don't use this color as background-color property. Level of energy-consuming in OLED's screen is depend at luminance of color.\n` +
+		`If you want to create more sustainable website use colors with lower luminance`,
+});
+
+const meta = {
+	url: 'http://ec0lint.com/features/background-color-validation',
+};
+
+/** @type {import('ec0lint-style').Rule} */
+const rule = (primary) => {
+	return (root, result) => {
+		const validOptions = validateOptions(result, ruleName, { actual: primary });
+
+		if (!validOptions) {
+			return;
```

Comment:
(minor thing) no need of brackets when there's one line inside

Reply:
I fixed this minor thing